pkgs/applications/version-management/git-and-tools/ghorg/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ghorg";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "gabrie30";
+    repo = "ghorg";
+    rev = "${version}";
+    sha256 = "0diwndkckv6fga45j9zngizycn5m71r67cziv0zrx6c66ssbj49w";
+  };
+
+  deleteVendor = true;
+  vendorSha256 = sha256:158r3mg69ag2yj1j1wdycz1wlilpi3s639b5xk3ak1gd1sij8vqy;
+
+  subPackages = [ "." ];
+
+  modSha256 = sha256:1lywbq8d4rrmzynh3dmncsy9hk0dxj157vj89fl2vl7vsmwwngp2;
+
+  preBuild = ''
+    export HOME=$TMPDIR
+    mkdir -p $HOME/.config/ghorg
+    cp sample-conf.yaml $HOME/.config/ghorg/conf.yaml
+  '';
+
+  buildFlagsArray = [ "-ldflags=-s -w -X main.version=${version}" ];
+
+  meta = with stdenv.lib; {
+    description = "ghorg allows you to quickly clone all of an orgs, or users repos into a single directory";
```

Don't cite the software's name in the short `description` string. Also, start it with a capital letter, and don't end with punctuation.

Indeed, I think the "about" description is perfect here:

```suggestion
    description = "Quickly clone an entire org/users repositories into one directory";
```

Also, if you like, add a `longDescription` containing some 

```suggestion
    longDescription = ''
      ghorg allows you to quickly clone all of an orgs, or users repos into a single 
      directory. This can be useful in many situations including
      
      - Searching an orgs/users codebase with ack, silver searcher, grep etc..
      - Bash scripting
      - Creating backups
      - Onboarding
      - Performing Audits
    '';
```